### DISCUSSION TITLE
new midi out filter mode

### DISCUSSION THREAD

**Discussion by enorrmann** [bangcorrupt/hacktribe#251] [Ideas]
factory firmware allows 2 options, short and short + program... but in some cases it would be nice to have everything filtered but note events, some start stop events mess with my old keyboards
doable ?

**Comment by bangcorrupt** [comment]
MIDI filtering and re-mapping is planned. I made some progress with Soft Thru on the `experimental` branch.

**Reply by enorrmann** [reply]
awesome
